src/controller/python/chip/yaml/runner.py

```diff
@@ -504,10 +504,71 @@ def run_action(self, dev_ctrl: ChipDeviceCtrl) -> _ActionResult:
             return _ActionResult(status=_ActionStatus.ERROR, response=None)
 
 
+class DiscoveryCommandAction(BaseAction):
+    """DiscoveryCommand::FindCommissionable implementation"""
+
+    def __init__(self, test_step):
+        super().__init__(test_step)
+
+        if test_step.command == 'FindCommissionable':
+            self.filterType = discovery.FilterType.NONE
+            self.filter = None
+        elif test_step.command == 'FindCommissionableByCommissioningMode':
+            # this is just a "_CM" subtype
+            self.filterType = discovery.FilterType.COMMISSIONING_MODE
+            self.filter = 0
```

Comment:
Is thre a reason this is not None like for the "FindCommissionable" case, just to make it clear that this is not looking for CM=0?

Reply:
Set back to none. I believe this was left over from debugging CM failures.